src/components/extras/MapContainer.vue

```diff
@@ -59,7 +59,7 @@ export default {
             mapCenter: 'map/getNewMapCenter',
             userLocation: 'map/getUserLocation',
         }),
-        ...mapState('map', ['userCurrentLocation']),
+        ...mapState('map', ['userCurrentLocation', 'filteredSrpResults']),
```

Comment:
since the map should not be tightly coupled  with SRP, we should rename it to `filteredSpots` or just `spots` and parent should have the filtering logic